content/influxdb/v2.0/reference/cli/influxd/upgrade.md

```diff
@@ -9,26 +9,50 @@ products: [oss]
 ---
 
 Use the `influxd upgrade` command to upgrade an instance of InfluxDB 1.x to InfluxDB 2.0.
-This command copies all data in [databases](/influxdb/v1.8/concepts/glossary/#database) and 
+This command copies all data in [databases](/influxdb/v1.8/concepts/glossary/#database) and
 [retention policies](/influxdb/v1.8/concepts/glossary/#retention-policy-rp) (used in 1.x)
 over to [buckets](/influxdb/v2.0/reference/glossary/#bucket) (used in 2.0).
 
 {{% warn %}}
 Be sure to back up all data before upgrading with `influx upgrade`.
 {{% /warn %}}
 
+This command performs the following actions:
+
+1. Reads 1.x configuration file and creates a 2.x config file with matching options.
+   Unsupported 1.x options are reported.
+2. Copies and upgrades 1.x database files.
+3. Creates a script for creating tokens that correspond to 1.x users.
+   This script needs to be revised and run manually after starting 2.x.
+
+If the configuration file is not available, the 1.x database folder can be passed via th `--v1-dir` flag.
+The target 2.x database directory is specified by the `--engine-path` option.
```

Comment:
Is this used in conjunction with passing the 1.x config file? If yes, does it make sense to combine both options?
Specify the target 2.x ...

Reply:
This is used if the user wishes to override the destination data path for the InfluxDB 2.x instance. Ordinarily, the default value of this argument would be fine.